db/migrate/20200831130746_create_active_storage_tables.active_storage.rb

```diff
@@ -0,0 +1,27 @@
+# This migration comes from active_storage (originally 20170806125915)
+class CreateActiveStorageTables < ActiveRecord::Migration[5.2]
+  def change
+    create_table :active_storage_blobs do |t|
+      t.string   :key,        null: false
+      t.string   :filename,   null: false
+      t.string   :content_type
+      t.text     :metadata
+      t.bigint   :byte_size,  null: false
+      t.string   :checksum,   null: false
+      t.datetime :created_at, null: false
+
+      t.index [ :key ], unique: true
+    end
+
+    create_table :active_storage_attachments do |t|
+      t.string     :name,     null: false
+      t.references :record,   null: false, polymorphic: true, index: false
+      t.references :blob,     null: false
+
+      t.datetime :created_at, null: false
+
+      t.index [ :record_type, :record_id, :name, :blob_id ], name: "index_active_storage_attachments_uniqueness", unique: true
+      t.foreign_key :active_storage_blobs, column: :blob_id
+    end
+  end
+end
```

Comment:
What are these tables for?

Reply:
Paperclip is the Gem we  use to manage images. It has been deprecated since rails 5. Rails 5 introduced ActiveStorage which provides a general mechanism for managing attachments.

It is our eventual intention to migrate from paperclip to active storage. This commit enabled the active storage feature and the regions features will attach images using active storage. However, I decided to package it up a little differently and attachments will come a little later to a couple of different models.